sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryToTableIT.java

```diff
@@ -291,8 +292,10 @@ public void testStandardQueryWithoutCustom() throws Exception {
   @Test
   public void testNewTypesQueryWithoutReshuffleWithCustom() throws Exception {
     this.setupNewTypesQueryTest();
-    this.options.setExperiments(
-        ImmutableList.of("enable_custom_bigquery_sink", "enable_custom_bigquery_source"));
+    List<String> experiments = new ArrayList<>();
+    experiments.add("enable_custom_bigquery_sink");
```

Comment:
Why did you need to make this change?

Reply:
If there is dataflowWorkerJar set, then we will append the experiment flag into experiments: https://github.com/apache/beam/blob/master/runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java#L744. In this case, if the experiments from pipeline is a immutable collection, then when trying to append the worker_jar experiment, an exception will be thrown.

Reply:
Wouldn't it make sense to fix the underlying bug to correctly concatenate two lists together instead of assuming the underlying list is mutable?